control-plane/stor-port/src/types/v0/store/snapshots/volume.rs

```diff
@@ -61,7 +61,7 @@ impl VolumeSnapshot {
                 self.spec().uuid(),
                 self.spec.source_id.to_string(),
                 self.metadata.prepare(),
-                "what's-my-name?",
+                self.spec.uuid.to_string(),
```

Comment:
`self.spec.uuid.to_string()` -> `self.spec().uuid().to_string()` ?

Thought: Should we rather pass the volume id as name  as name would be stored as meta in spdk and could be useful?

Reply:
I don't think we can because this is the bdev name for the snapshot, it has to be unique AIUI, if it's the volume name it would collide with the nexus?

Reply:
Right.